functions/go/enable-gcp-services/gcpservices/services.go

```diff
@@ -0,0 +1,55 @@
+package gcpservices
+
+import (
+	"fmt"
+	"strings"
+
+	"sigs.k8s.io/kustomize/kyaml/yaml"
+)
+
+// Service represents a valid GCP Service
+type Service struct {
+	APIVersion string `yaml:"apiVersion"`
+	Kind       string `yaml:"kind"`
+	Metadata   struct {
+		Name string `yaml:"name"`
+	} `yaml:"metadata"`
+	Spec struct {
+		ResourceID string `yaml:"resourceID"`
+		ProjectRef struct {
+			External string `yaml:"external,omitempty"`
+		} `yaml:"projectRef,omitempty"`
+	} `yaml:"spec"`
+}
+
+// getServicesList creates a slice of services from service strings
+func getServicesList(name string, inputServices []string, projectID string) ([]Service, error) {
```

Comment:
`getServicesList` doesn't seem to be accurate naming here. Maybe we can use `CreateServicesList` or `ConstructServicesList` or something more reasonable.

Reply:
done